pkg/tangy/interface.go

```diff
@@ -19,7 +19,7 @@ func New(dbConfig Database, logConfig Logger) (Tangy, error) {
 	if dbConfig.PoolLimit == 0 {
 		dbConfig.PoolLimit = DefaultMaxPoolLimit
 	}
-	pxConfig.MaxConns = int32(dbConfig.PoolLimit)
+	pxConfig.MaxConns = dbConfig.PoolLimit
```

Comment:
This works! but changing types here will also require changes in the backend. Let's fix this linting issue by checking the int max, like this: https://github.com/content-services/content-sources-backend/blob/93863b503e71c6bb93e10f1b2239887efc476c86/pkg/tasks/queue/pgqueue.go#L218-L220